src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java

```diff
@@ -34,8 +34,7 @@ public static void main(String[] args) {
      * @param weight - вес
      */
     static int calculateDeviationStandardWeight(int height, int weight) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+
+        return weight - (height - 110);
```

Comment:
"Code smell: Magic Number
Solution: Named Constants"

Reply:
this fixed